Classes/Event/EventTypeResolver.php

```diff
@@ -55,7 +55,7 @@ public function injectObjectManager(ObjectManagerInterface $objectManager): void
      */
     public function initializeObject(): void
     {
-        $this->mapping = self::eventTypeMapping($this->objectManager);
+        $this->mapping = static::eventTypeMapping($this->objectManager);
```

Comment:
Without this change the `CompileStatic` annotation is actually useles.. 😱

Reply:
haha :D good one!